src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1016,11 +1022,12 @@ def dbapi(
         lower_bound: Optional[Union[str, int]] = None,
         upper_bound: Optional[Union[str, int]] = None,
         num_partitions: Optional[int] = None,
-        predicates: Optional[List[str]] = None,
         *,
         max_workers: Optional[int] = None,
+        query_timeout: Optional[int] = 0,
```

Comment:
I suggest the following signature to separate positional parameters and key-word arguments (moving the 
"*" forward)

def dbapi(
    self,
    create_connection,
    table,
    *,
    column,
    ...
)